schema.sql

```diff
@@ -10,4 +10,43 @@ CREATE TABLE animals (
 );
 
 -- Add a new column "species" of type string to the table 'animals'.
-ALTER TABLE animals ADD COLUMN species VARCHAR(100);
\ No newline at end of file
+ALTER TABLE animals ADD COLUMN species VARCHAR(100);
+
+-- DAY 3 : create new table owners
+-- id: integer (set it as autoincremented PRIMARY KEY)
+-- full_name: string
+-- age: integer
+CREATE TABLE OWNERS (
+    id SERIAL PRIMARY KEY,
+    full_name VARCHAR(100),
+    age INTEGER
+);
+
+-- Create a table species
+-- id: integer (set it as autoincremented PRIMARY KEY)
+-- name: string
+CREATE TABLE SPECIES (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(100)
+)
```

Comment:
Kindly note that a table should have a primary key for the optimization of data from a database table so please add a primary key to your tables

Reply:
Hello @usorfaitheloho ,